c2pa-proposal.md

```diff
@@ -98,3 +98,104 @@ The only requirement for the Implicit Attestation to work within C2PA Eco-system
     4) If the signing key is revoked for any reason (e.g. due to expiry), then a new key to be provisioned, has to undergo the same "Explicit Attestation Protocol".
 
 Specific Explicit "Key Attestation" Protocol is out of scope of C2PA Specification.
+
+## Explicit Attestations
```

Comment:
I think that the advantages of explicit attestations are more modest than this.  I offer to rewrite later